opa-docs/docs/opa-entity-checker/example-entity-checker-policy.md

```diff
@@ -1,53 +1,90 @@
 # Example Entity Checker Policy
 
-This is an example policy for the OPA Entity Checker plugin. This policy is used to check if an entity has the correct metadata set. This could be used as a starting point for your own policies.
+This is an example policy for the OPA Entity Checker plugin. This policy is used to check if an entity has the correct metadata set. This could be used as a starting point for your own policies!
 
 ```rego
 package entity_checker
 
 import rego.v1
 
-# By default we assume the entity is bad :)
-default good_entity := false
+is_system_present if {
+    input.spec.system
+}
 
-# Its a good entity if there are no error violations
-good_entity if {
-	count({v | some v in violation; v.level == "error"}) == 0
+check contains {
```

Comment:
Calling it a check makes it completely incompatible with `conftest`, which closes some use-cases such as pipeline or pre-commit validation.